tests/parse/struct.wat

```diff
@@ -0,0 +1,26 @@
+(module
```

Comment:
These files I think should be fine to drop into `tests/regression/*` and they'll be parsed/encoded, and I think that should remove the need for a new `tests/parse.rs`

Reply:
Ah, I was concerned that because `wabt` doesn't support the GC proposal yet (or may slightly differ) that we would get errors here. But if `wabt` fails to handle the file we just ignore it, so we're good.